flags/flags.go

```diff
@@ -16,14 +17,21 @@ import (
 // A CompletionHandler is the type of function that our flags library uses to handle completions.
 type CompletionHandler func(parser *flags.Parser, items []flags.Completion)
 
+// AdditionalUsageInfo is the type of function that seeks out auxiliary options and adds them
+// to the end of usage info
+type AdditionalUsageInfo func(parser *flags.Parser, wr *bufio.Writer)
```

Comment:
I think this should be `func(parser *flags.Parser, wr io.Writer)` - the callback shouldn't specifically care what kind of writer it is.

Reply:
Fair point! Thanks :)